src/developer/web-api/tracker.md

```diff
@@ -2152,9 +2152,9 @@ as shown:
 
 Potential duplicates are records we work with in the data deduplication feature. Due to the nature of the deduplication feature, this API endpoint is somewhat restricted.
 
-A potential duplicate represents a single or pair of records which are suspected to be a duplicate.
+A potential duplicate represents a pair of records which are suspected to be a duplicate.
```

Comment:
Just for consistency I would always go with Potential Duplicate or potential duplicate